### DISCUSSION TITLE
Welcome to PBRVulkan Discussions!

### DISCUSSION THREAD

**Discussion by Zielon** [Zielon/PBRVulkan#46] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by polik-G** [comment]
I want to learn about your project
but i can't run it
![4IT1 YWA UQ}H~Y5J0U9R](https://user-images.githubusercontent.com/58029171/132451877-504dc9f1-23b7-42e3-9722-6455460142e9.png)
error code is E0304 and C2665
Error occurred at Mesh.cpp
i think this problem This error is caused by tiny_obj_loader.h
because there is no such a file in this project,i get it from others
![QQ截图20210908133911](https://user-images.githubusercontent.com/58029171/132452621-817b3bdd-d255-42b0-8752-a95e311369de.png)

**Reply by Zielon** [reply]
Hey, I see that you are using Visual Studio, so I assume that this is a Windows platform. Did you run the installation script? If you are using multiple vcpkg it is important to run `vcpkg.exe integrate install`  when you switch between projects.

**Comment by polik-G** [comment]
emmm.That problem was solved, but a new one came 
QWQ
![image](https://user-images.githubusercontent.com/58029171/133135354-f6804538-9c6e-49ce-87b6-7cecdd6202a9.png)

**Reply by Zielon** [reply]
And this happens after you run the installation script and then running https://github.com/Zielon/PBRVulkan/blob/master/PBRVulkan/PBRVulkan.sln project?

**Reply by polik-G** [reply]
yes,i follow you step but  i cant run it correctly
Maybe I made a mistake. I’m not a computer student, but I have a little code foundation and I’m very interested in rendering.
